services/datcore-adapter/Dockerfile

```diff
@@ -13,10 +13,10 @@ FROM python:${PYTHON_VERSION}-slim-buster as base
 
 LABEL maintainer=sanderegg
 
-RUN set -eux; \
-  apt-get update; \
-  apt-get install -y --no-install-recommends gosu; \
-  rm -rf /var/lib/apt/lists/*; \
+RUN set -eux && \
+  apt-get update && \
+  apt-get install -y gosu && \
```

Comment:
you removed the ```--no-install-recommends```, please put it back. these are recommendations for making the sleekest images...

Reply:
copy pasta always fails